src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.jsx

```diff
@@ -1,32 +1,49 @@
-import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import React from 'react';
 
 import HospitalizationPeriodView from '../components/HospitalizationPeriodView';
 
-import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
+// import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
 import { unemployabilityTitle } from '../content/unemployabilityFormIntro';
 
-const { hospitalizationHistory } = fullSchema.properties;
+// const { hospitalizationHistory } = fullSchema.properties;
+
+import tempSchema from '../config/schema'; // Will change to fullSchema when json schema approved
+
+const { hospitalizationHistory } = tempSchema.properties;
+
+const hospitalTitle = <h4>Hospitalization</h4>;
+
+const recordsDescription = (
+  <div>
+    <p>
+      If you received care from a private medical facility, we can get that
+      information for you. We‘ll ask you about that later in this process.
+    </p>
+    <p>
+      If you received care at a VA medical facility, we have access to those
+      records and can get them for you.
+    </p>
+  </div>
+);
 
 export const uiSchema = {
   unemployability: {
     'ui:title': unemployabilityTitle,
     hospitalizationHistory: {
-      'ui:title': 'Hospitalization',
-      'ui:description': 'Dates you were hospitalized?',
+      'ui:title': hospitalTitle,
       'ui:options': {
         itemName: 'Hospital',
         viewField: HospitalizationPeriodView,
         hideTitle: true,
       },
       items: {
-        hospitalizationDateRange: dateRangeUI(
-          'From',
-          'To',
-          'End of hospitalization must be after start of treatment',
-        ),
         hospitalName: {
           'ui:title': 'Name of hospital',
         },
+        hospitalizationDateRange: {
+          'ui:title':
+            'When were you in this hospital? Include the dates you were admitted and discharged for each stay.',
+        },
         hospitalAddress: {
           addressLine1: {
```

Comment:
should this use the `addressUI` ?

Reply:
i ran into a problem where addressUI uses street1 and street2. and I needed addressLine1 and addressLine2

Reply:
Ok, i'll fix this in my 8940 schema changes for vets-website